<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1048</width>
    <height>992</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>961</width>
      <height>301</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QTextBrowser" name="CodeBox"/>
     </item>
     <item>
      <widget class="QTextBrowser" name="ResultBox"/>
     </item>
    </layout>
   </widget>
   <widget class="QTextBrowser" name="SyntaxTree">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>360</y>
      <width>481</width>
      <height>401</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>780</y>
      <width>961</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="LoadButton">
       <property name="text">
        <string>载入代码（LOAD）</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="RunButton">
       <property name="text">
        <string>运行代码（RUN）</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="ClearButton">
       <property name="text">
        <string>清除代码（CLEAR）</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="DebugButton">
       <property name="text">
        <string>调试/调试（DEBUG/STEP）</string>
       </property>
      </widget>
     </item>
    </layout>
    <zorder>DebugButton</zorder>
    <zorder>LoadButton</zorder>
    <zorder>RunButton</zorder>
    <zorder>ClearButton</zorder>
   </widget>
   <widget class="QTextEdit" name="CommandInputBox">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>850</y>
      <width>961</width>
      <height>71</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>10</y>
      <width>72</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>代码</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>10</y>
      <width>72</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>运行结果</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>340</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>语句与语法树</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>830</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>命令载入窗口</string>
    </property>
   </widget>
   <widget class="QLabel" name="ErrorReminder">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>920</y>
      <width>921</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTextBrowser" name="VariableMap">
    <property name="geometry">
     <rect>
      <x>525</x>
      <y>361</y>
      <width>481</width>
      <height>401</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>340</y>
      <width>72</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>变量表</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1048</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuBASIC_interpreter">
    <property name="title">
     <string>BASIC interpreter</string>
    </property>
   </widget>
   <addaction name="menuBASIC_interpreter"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
